src/main/java/dukeproj/Storage.java

```diff
@@ -0,0 +1,114 @@
+package dukeproj;
+
+import dukeproj.data.Calender;
+import dukeproj.exception.BadDateException;
+import dukeproj.tasks.Deadline;
+import dukeproj.tasks.Event;
+import dukeproj.tasks.Task;
+import dukeproj.tasks.Todo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Represents the storage file for DukeProject.
+ */
+public class Storage {
+    private File file;
+
+    /**
+     * Prints the file in storage into an ArrayList.
+     * Takes in a calender to update the calender with the tasks in the file.
+     * @param calender calender to be updated by tasks in storage file.
+     * @return ArrayList made from tasks in storage file.
+     */
+    public ArrayList<Task> printFileIntoList(Calender calender) {
+        try {
+            Scanner sc = new Scanner(file);
+            ArrayList<Task> tasks = new ArrayList<>();
+            while (sc.hasNext()) {
+                String str = sc.nextLine();
+                String[] parts = str.split("\\|");
+                switch (parts[0]) {
+                case "T": {
```

Comment:
I don't think you need to put curly braces for [`case` statements](https://nus-cs2103-ay1920s2.github.io/website/coding-standards/java/intermediate.html#layout)...